### ISSUE TITLE
Open Source Mycetoma (MycetOS) Meeting Apr 27 2021

### ISSUE THREAD

**Issue by kym834** [OpenSourceMycetoma/Series-1-Fenarimols#50]
Hi All, 
Our next meeting is on April 27 and we'll be meeting at Zoom ID: 992-8154-9497 https://uni-sydney.zoom.us/j/99281549497.

**_Time to be confirmed (see #28)_**

Recording of this meeting to go here

Previous meeting are here (#47)

## Agenda

**1. Update on SAR analysis**
 - Do we have some leads on chemical space to follow from our analysis?
 - Collab with Kaitlin from Uni of Auckland (#49)

**2. Science updates from each site**
- UCL
- Sydney
- Emeritus

**3. Series 2 update**
- [Resynth of MMV006357](https://github.com/OpenSourceMycetoma/Series-2-Aminothiazoles/issues/4) (awaiting assignment of a MyOS number)

**4. Outreach and Comms**
- ISNTD Mycetoma event
- Newsletter

@OpenSourceMycetoma/corecontrib, please add anything you wish to discuss to the agenda

**Comment by wwjvdsande** [comment]
@kym834 is emeritus ErasmusMC?

**Comment by kym834** [comment]
Yeah, sorry! I couldn't remember the name 100% and just googled it haha. I'll change above :)

**Comment by MFernflower** [comment]
@dmitrij176 @kym834 RE: the oxime I proposed: 

![Ketoxim](https://user-images.githubusercontent.com/3164942/116238625-9761c280-a72f-11eb-83ea-34bfb40f3f9f.png)

**Comment by drc007** [comment]
You will get double bond isomers of the ovine.

**Comment by MFernflower** [comment]
Will need to be noted once compound is made as being a racemate of cis/trans

>

**Comment by MFernflower** [comment]
@dmitrij176 @drc007 @wwjvdsande Was the Pyrifenox that was screened racemic or enantiopure?

**Comment by mattodd** [comment]
"Racemate" isn't the right word here @MFernflower (nor are cis and trans, if we're being correct). I imagine the isomer ratio is known for lab-prepared Pyrifenox, but I wonder if the isomers interconvert in vivo, and I'm sure that must be known too?
I'm suddenly asking myself what the IUPAC word is for 1:1 mixture of E and Z isomers. Not that that would be a terribly common situation, but still.

**Comment by MFernflower** [comment]
It seems like commercially it's used as a mix of anti and syn isomers???

https://sor.epa.gov/sor_internet/registry/substreg/searchandretrieve/substancesearch/search.do?details=displayDetails&selectedSubstanceId=88140 


@mattodd

**Comment by dmitrij176** [comment]
I think its a mix as well, though there is no direct indication on that.

![Pyrifenox_Structural_Formula_V1 svg](https://user-images.githubusercontent.com/44785501/116435901-c406fe80-a843-11eb-8eea-351ee0954412.png)

**Comment by MFernflower** [comment]
@dmitrij176 Of course the compound I proposed depends on if @kym834 can/when get the O-methylhydroxylamine 


I did find an interesting patent possibly allowing for conversion to a single isomer

**Comment by MFernflower** [comment]
https://patents.google.com/patent/EP0009865A1/en

**Comment by MFernflower** [comment]
@dmitrij176 @kym834 I also dug up a interesting synthesis route: http://www.asianjournalofchemistry.co.in/User/ViewFreeArticle.aspx?ArticleID=19_3_81

**Comment by MFernflower** [comment]
@dmitrij176 Is this in refrence to pyrifenox or to the molecule I proposed?

**Comment by dmitrij176** [comment]
> @dmitrij176 Is this in refrence to pyrifenox or to the molecule I proposed?
pyrifenox

**Comment by MFernflower** [comment]
Might be interesting to have @kym834 do the bismuth grinding route than for the compound I proposed

**Comment by dmitrij176** [comment]
@MFernflower agree. I mean, at the moment pyrifenox is only the starting molecule in the oxime series. For other novel oximes, I would be interested in grinding too. Apparently this method gives good yields, so its worth trying.

**Comment by MFernflower** [comment]
@dmitrij176

This ketone is cheap, available in the uk and is only one step away from a
novel oxime

http://www.fluorochem.co.uk/Products/Search?searchType=N&searchText=3-(4-chlorobenzoyl)pyridine


Might be worth a shot

>

**Comment by mattodd** [comment]
Hi, sorry to raise this again, @OpenSourceMycetoma/corecontrib, but when's the next meeting? I'd like to invite along @AndyBishopSevenoaks .

As is now traditional, I have two Outlook bookings. 

![Screenshot 2021-05-12 at 11 31 56](https://user-images.githubusercontent.com/4386101/117961302-df7b0a80-b315-11eb-8d73-6b5c00f86a07.png)

Could we erase these and just sort the next meeting time immediately after each meeting? 

I'm happy, though not _delighted_, with 10pm London Tue 25th.
